learn/get-started/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Get Started
+---
+
+The best way to start your journey with Stellar is to create a [test network](./test-net.html) account and send your first transaction.
+
+In this tutorial we will be using our [JavaScript SDK](https://www.stellar.org/developers/js-stellar-sdk/learn/index.html) but you can use other [SDKs](https://www.stellar.org/developers/horizon/learn/#libraries) as well.
+
+## Installing `stellar-sdk`
+
+First create your NodeJS project if `package.json` file is not present in your working directory:
+```
+npm init
+```
+
+Then install our JavaScript SDK:
+```
+npm install --save stellar-sdk
+```
+
+## Creating a test network account
+
+First you have to generate your key pair.
+
+`generate_keypair.js` file:
+```js
+var StellarSdk = require('stellar-sdk');
+var keypair = StellarSdk.Keypair.random();
+
+console.log('Account ID:');
+console.log(keypair.accountId());
+console.log('Secret Seed:');
+console.log(keypair.seed());
+```
+
+Now, let's execute the file:
+```
+$ node generate_keypair.js
+
+Account ID:
+GDGOKHIRX63EIAXKVI77BQV7LMDUH7DR4BMDDU77DJUXLPUU5HAXGN64
+Secret Seed:
+SD5ALH5UA7LRMN6ZB2QILW35YZTEOE37M3JVNAKZ3KWT7QSC77ABJ3ID
+```
+
+You will use your key pair to receive funds (account ID) and to sign your transaction (secret seed). You can share your account ID publicly but you must keep your seed secret. Otherwise, someone may steel your funds!
```

Comment:
Fixed in https://github.com/stellar/docs/commit/890632e7bbb5c52511a76a8ca70351e9c1459e68